tools/sctool/src/main/java/com/horizen/ScBootstrappingToolCommandProcessor.java

```diff
@@ -506,12 +504,12 @@ private void printGenesisInfoUsageMsg(String error) {
                       "\t\t\"updateconfig\": boolean - Optional. Default false. If true, put the results in a copy of source config.\n" +
                       "\t\t\"sourceconfig\": <path to in config file> - expected if 'updateconfig' = true.\n" +
                       "\t\t\"resultconfig\": <path to out config file> - expected if 'updateconfig' = true.\n" +
-                      "\t\t\"blockversion\": int - Optional, default = 1. UTXO model block version=1, Account model block version=2.\n" +
+                      "\t\t\"model\": String - Optional, default = 'utxo', 'account' model.\n" +
```

Comment:
we need to update STF tests to start using `model` instead of `blockversion`.
@akrckn do you want to do it by yourself?

Reply:
Yes, updated the tests here, but still running - I'll resolve the conversation and re-request after all tests are executed.